CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 # Changelog for the Mapbox Maps SDK for Android
 
-Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
+Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
+
+## 9.1.0
+This beta release includes a feature with a known issue: the `within` expression evaluates point features inconsistently across zoom levels if the point lies near the boundary of a GeoJSON object ([#16301](https://github.com/mapbox/mapbox-gl-native/issues/16301])).
+### Features
+ - Added `within` expression for testing whether an evaluated feature lies within a given GeoJSON object. `within` accepts `Feature`, `FeatureCollection`, `Polygon`, and `MultiPolygon` as valid GeoJSON objects. ([#198](https://github.com/mapbox/mapbox-gl-native-android/pull/198))
+ - Added `in` expression for testing whether an item exists in an array or a substring exists in a string. ([#171](https://github.com/mapbox/mapbox-gl-native-android/pull/171))
+ - Added `FillLayer.getFillSortKey` and `LineLayer.getLineSortKey` methods to get the sort key of a layer.  Added `PropertyFactory.fillSortKey` and `PropertyFactory.lineSortKey` to set the sort key of a layer. Features with a higher sort key will appear above features with a lower sort key. ([#209](https://github.com/mapbox/mapbox-gl-native-android/pull/209))
```

Comment:
A few comments and questions:
- sort keys are applied to *features on a layer* rather than a layer itself
- could you confirm that it’s possible to set the sort key of features on a layer at runtime? 
- would you mind explaining what `PropertyFactory` is? I’ve been confused before about why I didn’t see `set*` APIs for style properties alongside their `get*` equivalents so this is probably a key piece of information I’m missing about Android

Reply:
`PropertyFactory` provides static methods to get `PropertyValue` which will be set into layers. For example we could `PropertyFactory.lineSortKey` to get the expression to set a layer's sort key in runtime.
```
layer.setProperties(lineSortKey(expression));
```